<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgQrCode</class>
 <widget class="QDialog" name="DlgQrCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="leAppendQr">
     <property name="placeholderText">
      <string>Emarks</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="C5LineEdit" name="leQty">
     <property name="placeholderText">
      <string>Quantity</string>
     </property>
     <property name="Type" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbName">
     <property name="text">
      <string>QR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="btnEdit">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/res.qrc">
       <normaloff>:/constructor.png</normaloff>:/constructor.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableWidget" name="tbl">
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>QR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remain</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>O</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="btnSave">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/res.qrc">
       <normaloff>:/save.png</normaloff>:/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="btnClose">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
